crates/cranelift/src/obj.rs

```diff
@@ -95,7 +95,9 @@ impl<'a> ModuleTextBuilder<'a> {
         func: &'a CompiledFunction,
     ) -> (SymbolId, Range<u64>) {
         let body_len = func.body.len() as u64;
-        let off = self.text.append(labeled, &func.body, None);
+        let off = self
+            .text
+            .append(labeled, &func.body, Some(self.isa.function_alignment()));
```

Comment:
I think this may have actually been the only place that `None` was passed in, so with this changing would it be possible to switch the alignment parameter to `u32` instead of `Option<u32>`?